.github/workflows/release.yaml

```diff
@@ -72,7 +72,7 @@ jobs:
     - name: Install pip dependencies
       if: steps.cache.outputs.cache-hit != 'true'
       run: |
-        pip install .[docs,tests] planetary_computer pystac pytest-rerunfailures tensorboard
+        pip install .[docs,tests] planetary_computer pystac pytest-rerunfailures
```

Comment:
The rest of these are things that we could subsume into `[tests]`, but they're also dependencies that I would like to remove in the future, and I don't think they're worth tracking.